src/missiondef.cpp

```diff
@@ -583,7 +583,7 @@ MISSION(_("Find Flag"), MGOAL_FIND_ITEM, 2, 100000, false,
          &mission_place::always, &mission_start::ranch_bartender_4,
          &mission_end::standard, &mission_fail::standard);
   ORIGINS(ORIGIN_SECONDARY);
-  ITEM("55gal_drum");
+  ITEM("metal_bigdrum");
```

Comment:
This may not update existing missions, the `item_id` is copied into the `mission` object when it's created and is later stored with the world data. Players that have the mission already will probably still have to collect items of the old type, which they can not do. (Not tested, my work just fine.)